Testscripts/Linux/perf_lagscope.sh

```diff
@@ -101,9 +107,10 @@ fi
 
 LogMsg "Now running Lagscope test"
 LogMsg "Starting server."
-ssh root@"${server}" "${cmd} -r" &
-sleep 20
+ssh root@"${server}" "timeout 230 ${cmd} -r${testServerIP}" &
+sleep 10
 LogMsg "lagscope client running..."
-ssh root@"${client}" "${cmd} -s${server} -i0 -n${pingIteration} -H > lagscope-n${pingIteration}-output.txt"
+ssh root@"${client}" "timeout 220 ${cmd} -s${testServerIP} -i0 -n${pingIteration} -H -P -a40 -l1 -c98> lagscope-n${pingIteration}-output.txt"
+sleep 220
```

Comment:
the command run on client will last 220, no need extra sleep?

Reply:
and could you please add comments for "-P -a40 -l1 -c98"?

Reply:
ssh root@"${client}" "timeout 220 ${cmd} -s${testServerIP} -i0 -n${pingIteration} -H -P -a40 -l1 -c98 > lagscope-n${pingIteration}-output.txt"

Reply:
fixed